README.md

```diff
@@ -73,6 +73,8 @@ a template to create your ``.env`` file. For Bootcamp to work, it needs 6 values
 
     The URL of your bootcamp instance on Docker (``http://<BOOTCAMP_DOCKER_MACHINE_IP>:8099``)
 
+- ``FACEBOOK_PIXEL_ID`` - Log into Facebook and create one at https://www.facebook.com/events_manager/pixel/
```

Comment:
We have the devops FB account now. We should probably mention that as option 1, and suggest creating one's own pixel as a fallback